uber/utils.py

```diff
@@ -285,3 +285,19 @@ def mount_site_sections(module_root):
     for section in sections:
         module = importlib.import_module(basename(module_root) + '.site_sections.' + section)
         setattr(Root, section, module.Root())
+
+
+def build_uber_absolute_url(relative_uber_page_url):
```

Comment:
I, uh, don't really see how this function does anything useful...? Our need for absolute URLs doesn't change based on any dynamic logic that I know about.